acid/acid3/test.html

```diff
@@ -2264,7 +2264,7 @@
       assert(test.htmlFor != test.getAttribute('htmlFor'), "pots: htmlFor attribute equals htmlFor property");
       assert(!('for' in test), "pots: element has for property");
       test['for'] = "oil";
-      assert(test.htmlFor != "oil", "pots: for property affected htmlFor");
+      assert(test.htmlFor == "oil", "pots: for property did not affect htmlFor");
```

Comment:
I believe the comment should not be changed, because previously the comment mismatched the test, and now that the test is changed the old comment is correct.

Reply:
I think this comment (and others in this test) are meant to be read more as error messages, since they'll show up when the test fails, telling you why it failed. (they're in the past tense, stating what happened, not what is expected)
Changing it back will make it inconsistent with how the comments in this test are generally written, which I don't think is a good thing.
And I'd rather not lump "reword all the test comments in Acid3" into this PR as an unrelated change.

Reply:
Yes, I see what you mean. So no need to change it and I think things are good as they seem.